modules/integration_azure-storage-account-blob/conf/readme.yaml

```diff
@@ -0,0 +1,6 @@
+documentations:
+  - name: Azure Monitor metrics
+    url: 'https://docs.microsoft.com/en-us/azure/azure-monitor/platform/metrics-supported'
+source_doc: >-
+  We are using metrics from the
+  [Microsoft.Storage/storageAccounts](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/metrics-supported#microsoftclassicstoragestorageaccountsblobservices) namespace.
```

Comment:
I would add a note about disabled detectors (why disabled by default and when to use them)